independent-publisher-connectors/OpenAI/OpenAI.swagger.json

```diff
@@ -0,0 +1,261 @@
+                      },
+                      "logprobs": {
+                        "type": "string",
+                        "description": "logprobs",
+                        "title": "Logprobs"
+                      },
+                      "finish_reason": {
+                        "title": "Finish reason",
+                        "type": "string",
+                        "description": "Reason why the text finished (stop condition / natural end / length)"
+                      }
+                    }
+                  },
+                  "description": "Returned Completion(s)"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Completion",
+        "description": "GPT3 Completes your prompt",
+        "operationId": "Completion",
+        "parameters": [
+          {
+            "name": "engine",
+            "in": "path",
+            "type": "string",
+            "description": "The used engine, choose between text-davinci-002, text-curie-001, text-babbage-001, text-ada-001",
+            "required": true,
+            "default": "text-davinci-002",
+            "x-ms-visibility": "important",
+            "x-ms-summary": "Engine",
+            "enum": [
+              "text-davinci-002",
+              "text-curie-001",
+              "text-babbage-001",
+              "text-ada-001"
+            ],
+            "x-ms-enum-values": [
+              {
+                "displayName": "DaVinci - most expensive",
+                "value": "text-davinci-002"
+              },
+              {
+                "displayName": "Curie",
+                "value": "text-curie-001"
+              },
+              {
+                "displayName": "Babbage",
+                "value": "text-babbage-001"
+              },
+              {
+                "displayName": "Ada - cheapest",
+                "value": "text-ada-001"
+              }
+            ]
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "prompt": {
+                  "type": "string",
+                  "description": "Text that will be completed by GPT3",
+                  "title": "prompt",
+                  "default": "What is your favorite animal and why? Tell me also about the size and weight of this animal."
+                },
+                "n": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "How many completions to generate for each prompt",
+                  "default": 1
+                },
+                "best_of": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "Generates best_of completions server-side and returns the \"best\". Must be greater than \"n\". Use with caution, can consume a lot of tokens.",
```

Comment:
@PowerRobin `"best of"` ?

Reply:
Changed to:
If set to more than 1, generates multiple completions server-side and returns the \"best\".